cylc/flow/post_install/__init__.py

```diff
@@ -0,0 +1,31 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Plugins for running Python code after installation of a workflow.
+
+Built In Plugins
+----------------
+
+Cylc Flow provides the following post-install plugins:
```

Comment:
We can document the pre-install and post-install entry points together as "install plugins" since they have related functionality and plugins may use both (e.g. `cylc-rose`).